### ISSUE TITLE
If map is included with distribution, but still in mod center, MG may crash at game mods menu

### ISSUE THREAD

**Issue by andy5995** [MegaGlest/megaglest-source#161]
Was meteor added to the 3.13.0 distribution?

For me and @tomreyn, MG crashes when we go into "game mods", click the meteor map, then return to the main menu.

I believe this is because MG is looking for the map in ~/.megaglest/maps
If it doesn't find it there, it crashes.

Now it's included in the distribution maps directory, but still apparently in the mod center.

The crash will not happen if a copy of meteor.mgm is placed in ~/.megaglest/maps

**Comment by tomreyn** [comment]
Indeed, this map appears to be included in the distribution now, and thus needs to be removed off the mod center for this release (only?). 

The mod center database contains [a field for minimum game version](https://github.com/MegaGlest/megaglest-masterserver/blob/615d10c32e53dda1b4a1bb4ac0fcdcc125641226/install/scheme_mysql.sql#L62) (by which mods will start to show up on mod center), but this issue suggests we may also need to have a maximum version (for when we add files previously available on the mod center into the distribution).

Additionally, we should add safeguards to handle this situation (a mod is included into distribution but still listed on mod db) gracefully in the future.

For now, I suggest we remove this mod off the mod DB, which fixes it for the current release, and makes the mods unavailable in past releases (where it was not part of the distribution). 

We also need to check whether this also affects other mods.

**Comment by andy5995** [comment]
The map cliff_valley is also affected by this

**Comment by titiger** [comment]
Terminal output including stacktrace:
http://pastebin.com/raw/CuiDS22P

**Comment by titiger** [comment]
I will remove critical maps now from mod center.

**Comment by andy5995** [comment]
It actually seems the 2 maps mentioned were included since 3.12.0 and not recently added to 3.13.

https://github.com/MegaGlest/megaglest-data/tree/3.12.0/maps

**Comment by andy5995** [comment]
"prevention" was the 3rd and last map. @titiger removed all 3.
